docs/docsource/Reference/bucket_lifecycle_operations/put_bucket_lifecycle.rst

```diff
@@ -592,7 +589,7 @@ The following lifecycle configuration specifies two rules, each with one
 action.
 
 -  The Transition action specifies objects with the “documents/” prefix
-      to transition to the GLACIER storage class 30 days after creation.
+      to transition to the Wasbi_cloud storage class 30 days after creation.
```

Comment:
Wasbi_cloud -> wasabi_cloud

Reply:
Whoops, I guess you use capitalized Wasabi from here down (in the example above it's lowercase). Either way, doesn't really matter

Reply:
I probably should put that second "a" in "Wasbi" too...